packages/Maleo.js/src/build/static/static.ts

```diff
@@ -0,0 +1,39 @@
+// tslint:disable:no-console
+import fs from 'fs';
+import path from 'path';
+import { StaticPages } from '@interfaces/build/IWebpackInterfaces';
+import { render } from '@server/render';
+import { BUILD_DIR, STATIC_BUILD_DIR } from '@constants/index';
+
+/**
+ * acquire corresponding pages which want to be exported as static html
+ * render App, Wrap, & Document for each page
+ * renderToString & dump it to file
+ */
+export const buildStatic = async (staticPages: StaticPages, dir: string) => {
+  const cwd: string = path.resolve(dir);
+  const assetDir: string = path.resolve(cwd, BUILD_DIR, 'client');
+  const staticDir: string = path.resolve(cwd, STATIC_BUILD_DIR);
+  if (!fs.existsSync(staticDir)) {
+    fs.mkdirSync(staticDir);
+  }
+
+  Object.keys(staticPages).map(async (p) => {
+    const html = await renderStatic(p, assetDir);
+    const pageName = p.substring(1, p.length);
```

Comment:
will it be better to use RegEx instead?
to prevent user's mistake if they write a valid url like this one `//route-1`
```javascript
// ....
const pageName = p.replace(/\/+(.+)$/, '$1');
// ...
```